Data/System/strings.csv

```diff
@@ -457,11 +458,11 @@ Turn Left,À gauche,Nach links,,Sinistra,Gå åt vänster,
 Turn Right,À droite,Nach rechts,,Destra,Gå åt höger,
 Forward,En avant,Vorwärts,,Avanti,Gå framåt,
 Backward,En arrière,Rückwärts,,Indietro,Gå bakåt,
-Kick,Coup de pied,Treten,Patada,Calcio,Sparka,Schoppen
-Pickup/Drop,Ramasser/Lâcher,Aufheben/Fallenlassen,Coger/Soltar,Prendi,Plocka upp/Släppa,Oppakken/Loslaten
-Auto-Walk,Marcher en continu,Automatisch laufen,Caminar auto.,Camminare auto.,Autogå,Auto-loop
+Kick,Coup de pied,Treten,Patada,Calcia,Sparka,Schoppen
+Pickup/Drop,Ramasser/Lâcher,Aufheben/Fallenlassen,Coger/Soltar,Raccogli/Lascia,Plocka upp/Släppa,Oppakken/Loslaten
```

Comment:
If we change the terminology here, I suggest we also change it in the dialog text for consistency. Look for "pickup" in English ("See if you can lift the bar by pressing your Pickup button" and "To drop a bomb, press the Pickup/Drop button")